tests/test_vector2_ctor.py

```diff
@@ -38,3 +40,9 @@ def test_ctor_noncopy_subclass():
 def test_ctor_noncopy_superclass():
     v = Vector2(1, 2)
     assert V(v) is not v
+
+
+@given(v=vectors())
+def test_ctor_copy(v: Vector2):
+    """Test that Vector2 instances can be copied."""
```

Comment:
Docstring and function name need to reflect what the test does.

Also, is `test_vector2_ctor.py` the right location for tests of a protocol that isn't the constructor?

Reply:
@astronouth7303 You didn't reply, re: the choice of file for testing the pickle protocol.

Reply:
It has more to do with object instantiation than any operation.

Reply:
M'kay.